.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: |
+          npm install
```

Comment:
Npm?

Reply:
Yeah I can't get `yarn link` working - happy if you want to have a crack at it

Reply:
I think yarn had some issues to get package linking working across host and docker container. Same issues were seen with older npm versions: https://github.com/yannickcr/eslint-plugin-react/pull/3120#issuecomment-959089250.